core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -35,6 +35,24 @@ package object config {
   private[spark] val SPARK_EXECUTOR_PREFIX = "spark.executor"
   private[spark] val SPARK_TASK_PREFIX = "spark.task"
 
+  private[spark] val SPARK_RESOURCES_COORDINATE =
+    ConfigBuilder("spark.resources.coordinate.enable")
+      .doc("Whether to coordinate resources automatically among workers/drivers(client only) " +
+        "in Standalone. If not, user should be responsible for assigning different resources " +
```

Comment:
nit, reword a bit.  

If false, the user is responsible for configuring different resources for workers/drivers that run on the same host.